app/templates/components/project-task-list.hbs

```diff
@@ -1,67 +0,0 @@
-<div class="tasks-menu">
-  <div class="left">
-    {{task-filter-type selectedTypes=selectedTypes filterByType="filterByType"}}
-  </div>
-  <div class="left statuses">
-    <ul>
-      <li>
-        <a class="open {{if isFilteringOpenTasks "active"}}" {{action "filterByStatus" "open"}}>{{project.openTasksCount}} Open</a>
-      </li>
-      <li>
-        <a class="closed {{if isFilteringClosedTasks "active"}}" {{action "filterByStatus" "closed"}}>{{project.closedTasksCount}} Closed</a>
-      </li>
-    </ul>
-  </div>
-  <div class="right">
-    {{link-to 'New task' 'project.tasks.new' tagName='button' class="default small new-task"}}
-  </div>
```

Comment:
The test failures are isolated to this button being gone. I think we need to add the `tasks-menu` back in, but put this button under `<div class="left">`. That should fix it.